source/reference/methods.txt

```diff
@@ -356,6 +356,56 @@ Connection Methods
 
      - Description
 
+   * - :method:`Mongo()`
+
+     - JavaScript constructor to instantiate a database connection from
+       the ``mongo`` shell or from a JavaScript file.
+
+       The :method:`Mongo()` method has the following parameters:
+
+       .. list-table::
+           :header-rows: 1
+           :widths: 20 20 80
+
+           * - Parameter
+       
+             - Type
+       
+             - Description
+       
+           * - ``host``
+       
+             - string
+       
+             - *Optional*
+             
+               The host, either in the form of ``<host>`` or 
+               ``<host><:port>``.
```

Comment:
Fwiw, this can be any valid mongodb connection string, and include e.g. database name as well. I think this applies to the old shell as well.